apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -221,6 +227,15 @@ const BookingPage = ({
         collectPageParameters("/book", { isTeamBooking: document.URL.includes("team/") })
       );
     }
+    reserveSlot();
+    const interval = setInterval(
+      reserveSlot,
+      parseInt(process.env.NEXT_PUBLIC_MINUTES_TO_BOOK || "5") * 60 * 1000 - 2000
```

Comment:
We can abstract it into constants.ts like other env variables so that the default value is consistently set.